lib/translator/BUILD

```diff
@@ -1,3 +1,17 @@
+cc_library(
+  name = "inquiry_lib",
+  srcs = ["inquiry.cc", "common.cc"],
+  hdrs = ["inquiry.h", "common.h"],
```

Comment:
We should have `": common"` in the dependencies rather than the hdrs/srcs.

Reply:
we can remove them from the hdrs/srcs now that they're in the deps

Reply:
Still needs to be addressed